not working correctly

case sensitive for "PostBox" and "React":
Error: Cannot find module './components/post/PostBox' from '/home/projects/keanux-personal/app'
Error: Cannot find module 'React' from '/home/projects/keanux-personal/app/components/post'

--- a/app/components/post/postlist.js
+++ b/app/components/post/postlist.js
@@ -1,5 +1,5 @@
 // Required Library
-var React = require('React');
+var React = require('react');
 
 // Related Control
 var Post = require('./post');

--- a/app/components/post/postmeta.js
+++ b/app/components/post/postmeta.js
@@ -1,5 +1,5 @@
 // Required Library
-var React = require('React');
+var React = require('react');
 var Ago = require('react-ago-component');
 
 // Control

--- a/app/main.js
+++ b/app/main.js
@@ -2,7 +2,7 @@
 var React = require('react');
 
 // Related Control
-var PostBox = require('./components/post/PostBox');
+var PostBox = require('./components/post/postbox');
 
 React.render(
     <PostBox url="/api/posts" />,

